.github/workflows/generate-edm4hep-file.yaml

```diff
@@ -0,0 +1,45 @@
+name: Generate an EDM4Hep file
```

Comment:
Can we not attach this to the usual tests we run? Resp. what am I missing that speaks against this? File size, run time,...?

Reply:
Changed now to a test